### PR TITLE
Turned the header image in readme.md into a link.

### PR DESCRIPTION
**Pull request by Foaly**
I've always found it irritating, that the header logo in readme.md is not a link to the website. When changing it I discovered a bunch of http links, so I changed them all to https. For the smaller website where I didn't know if they support it, I've checked.

### CODE REVIEW

File: doc/footer.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
         </div>
         <div id="footer-container">
             <div id="footer">
-                SFML is licensed under the terms and conditions of the <a href="http://www.sfml-dev.org/license.php">zlib/png license</a>.<br>
+                SFML is licensed under the terms and conditions of the <a href="https://www.sfml-dev.org/license.php">zlib/png license</a>.<br>
                 Copyright &copy; Laurent Gomila &nbsp;::&nbsp;
-                Documentation generated by <a href="http://www.doxygen.org/" title="doxygen website">doxygen</a> &nbsp;::&nbsp;
+                Documentation generated by <a href="https://www.stack.nl/~dimitri/doxygen/" title="doxygen website">doxygen</a> &nbsp;::&nbsp;
```
Comment (eXpl0it3r) [inline doc/footer.html]: I would keep this. I mean HTTP links aren't really any issue and who knows if that weird URL will move one day, while doxygen.org is more likely to stay.

File: readme.md [hunk 2]
```diff
@@ -19,21 +19,21 @@ SFML is a simple, fast, cross-platform and object-oriented multimedia API. It pr
 
 ## Download
 
-You can get the latest official release on [SFML's website](http://www.sfml-dev.org/download.php). You can also get the current development version from the [Git repository](https://github.com/SFML/SFML).
+You can get the latest official release on [SFML's website](https://www.sfml-dev.org/download.php). You can also get the current development version from the [Git repository](httpss://github.com/SFML/SFML).
```
Comment (eXpl0it3r) [inline readme.md]: Interesting protocol you got there. 😉

File: src/SFML/Network/IpAddress.cpp [hunk 3]
```diff
@@ -145,7 +145,7 @@ IpAddress IpAddress::getPublicAddress(Time timeout)
 {
     // The trick here is more complicated, because the only way
     // to get our public IP address is to get it from a distant computer.
-    // Here we get the web page from http://www.sfml-dev.org/ip-provider.php
+    // Here we get the web page from https://www.sfml-dev.org/ip-provider.php
```
Comment (eXpl0it3r) [inline src/SFML/Network/IpAddress.cpp]: This can't be changed! 🙀

File: src/SFML/Window/Win32/WglExtensions.txt [hunk 4]
```diff
@@ -10,4 +10,4 @@ WGL_ARB_multisample
 WGL_ARB_pixel_format
 WGL_ARB_pbuffer
 WGL_ARB_create_context
-WGL_ARB_create_context_profile
\ No newline at end of file
+WGL_ARB_create_context_profile
```
Comment (eXpl0it3r) [inline src/SFML/Window/Win32/WglExtensions.txt]: What is this change ❔

File: doc/doxyfile.in [hunk 5]
```diff
@@ -20,7 +20,7 @@
 # This tag specifies the encoding used for all characters in the config file
 # that follow. The default is UTF-8 which is also the encoding used for all text
 # before the first occurrence of this tag. Doxygen uses libiconv (or the iconv
-# built into libc) for the transcoding. See http://www.gnu.org/software/libiconv
+# built into libc) for the transcoding. See https://www.gnu.org/software/libiconv
```
Comment (eXpl0it3r) [inline doc/doxyfile.in]: This is an auto-generated file, I wouldn't really edit it, but I guess it doesn't hurt either, so lets keep the changes.

File: readme.md [hunk 6]
```diff
@@ -19,21 +19,21 @@ SFML is a simple, fast, cross-platform and object-oriented multimedia API. It pr
...
 
 There are several places to learn SFML:
 
-  * The [official tutorials](http://www.sfml-dev.org/tutorials/)
-  * The [online API documentation](http://www.sfml-dev.org/documentation/)
-  * The [community wiki](https://github.com/SFML/SFML/wiki/)
-  * The [community forum](http://en.sfml-dev.org/forums/) ([French](http://fr.sfml-dev.org/forums/))
+  * The [official tutorials](https://www.sfml-dev.org/tutorials/)
+  * The [online API documentation](https://www.sfml-dev.org/documentation/)
+  * The [community wiki](httpss://github.com/SFML/SFML/wiki/)
```
Comment (eXpl0it3r) [inline readme.md]: Is this the super secure protocol? 😉

File: doc/footer.html (hunk 1 above)
Comment (Foaly) [reply doc/footer.html]: Makes sense I guess.

File: readme.md (hunk 2 above)
Comment (Foaly) [reply readme.md]: I ran one command twice... I knew I would miss something 😄

File: readme.md (hunk 6 above)
Comment (Foaly) [reply readme.md]: Yep http super secure is the new telnet!

File: src/SFML/Network/IpAddress.cpp (hunk 3 above)
Comment (Foaly) [reply src/SFML/Network/IpAddress.cpp]: Why not? the site is also reachable with https?

File: src/SFML/Window/Win32/WglExtensions.txt (hunk 4 above)
Comment (Foaly) [reply src/SFML/Window/Win32/WglExtensions.txt]: Newline at end of file.

File: src/SFML/Network/IpAddress.cpp (hunk 3 above)
Comment (LaurentGomila) [reply src/SFML/Network/IpAddress.cpp]: `sf::TcpSocket` is unable to do SSL.

**Comment by Foaly** [comment]
I will send an update later tonight!

File: src/SFML/Network/IpAddress.cpp (hunk 3 above)
Comment (Foaly) [reply src/SFML/Network/IpAddress.cpp]: Yep I mistook it for a documentation in a header.

File: doc/footer.html (hunk 1 above)
Comment (Foaly) [reply doc/footer.html]: http://www.doxygen.org/ is a redirect to that dutch URL though.

**Comment by Foaly** [comment]
Updated!

File: doc/footer.html (hunk 1 above)
Comment (eXpl0it3r) [reply doc/footer.html]: I know and? As I said, doxygen.org is more likely to stay than that Dutch URL.

Comment (Foaly) [reply doc/footer.html]: Yeah probably right!

**Comment by eXpl0it3r** [comment]
Merged in 989367e34552c39c999a36d12608bd11eaacc3d7

**Pull request closed without merging**